test/run_test.py

```diff
@@ -909,6 +910,11 @@ def get_selected_tests(options):
             WINDOWS_BLOCKLIST.append("jit")
             WINDOWS_BLOCKLIST.append("jit_fuser")
 
+        if distutils.version.LooseVersion(torch.version.cuda) >= "11.5":
```

Comment:
Is there an issue tracking why this is the case? Could you add a comment above here explaining why this has to be/linking to the issue?

Reply:
Yes totally The is the issue https://github.com/pytorch/pytorch/issues/69460 Will add a comment as well.